cypress/e2e/communityTemplates.test.ts

```diff
@@ -72,6 +80,37 @@ describe('Community Templates', () => {
       )
     })
 
+    it('can install using the keyboard "Enter" key and without clicking "Lookup Template"', () => {
```

Comment:
@TCL735 thank you for adding tests to this!

@kristinarobinson another good candidate for assertions that might be better suited in integration or unit tests